### DISCUSSION TITLE
tsserver stdio example

### DISCUSSION THREAD

**Discussion by m-anish-gupta** [typescript-language-server/typescript-language-server#857] [Q&A]
Can anyone share a sample on how to interact with the tsserver in stdio mode?

In my setup, i'm using a python wrapper on top of this which starts a async process in the background to launch tsserver in stdio mode using `typescript-language-server --stdio` command.

I constructed the `initialize` request params by reading the doc [here](https://github.com/typescript-language-server/typescript-language-server/blob/master/docs/configuration.md#initializationoptions) but when I try to resolve for symbol definitions I'm observing that the symbol is only resolved to the import statement and not to the specific file.

Any thoughts on this?

Also, is there any specific path where it's logs will be dumped?

**Comment by rchl** [comment]
I would recommend that you set it up to run in an editor of your choice, enable LSP logging there and figure out what to call based on its logs.
